modules/apps/27-interchain-accounts/host/keeper/keeper.go

```diff
@@ -36,7 +42,11 @@ type Keeper struct {
 
 	scopedKeeper exported.ScopedKeeper
 
-	msgRouter icatypes.MessageRouter
+	msgRouter   icatypes.MessageRouter
+	queryRouter icatypes.QueryRouter
+
+	// whitelist of module safe queries
+	mqsWhitelist []string
```

Comment:
quick comment, can we go with `AllowList` here to maintain symmetry with other naming conventions? :)

Reply:
done